lib/fetchers.nix

```
@@ -14,4 +14,60 @@
     "NIX_SSL_CERT_FILE"
   ];
 
+  /**
+    Converts an attrset containing one of `hash`, `sha256` or `sha512`, into one containing `outputHash{,Algo}`.
+
+    All other attributes in the set remain as-is.
+  */
+  normalizeHashes = {
+    hashes ? [ "sha256" "sha512" ]
+  }: args:
+    with builtins; with lib;
+    let
+      hNames = [ "hash" ] ++ hashes;
+      hAttrs = genAttrs hNames (const {});
```

```suggestion
    hashTypes ? [ "sri" "sha256" "sha512" ]
  }: args:
    with builtins; with lib;
    let
      hAttrs = genAttrs hashTypes (const {});
```

Maybe better? (I changed "hash" to "sri")

---

Oh, I got it, it's compatible with current `"hash"` attribute.